src/accounts/AccountPage.tsx

```diff
@@ -35,24 +60,45 @@ const AccountAboutPage: FC = () => {
     setSwitchAccountVisible(false)
   }
 
+  const inputStyle = {width: 250}
+  const labelStyle = {marginBottom: 8}
+
   return (
     <Page titleTag={pageTitleSuffixer(['About', 'Account'])}>
       <AccountTabContainer activeTab="about">
         <>
           {userAccounts && userAccounts.length >= 2 && (
-            <Button
-              text="Switch Account"
-              icon={IconFont.Switch_New}
-              onClick={showSwitchAccountDialog}
-              testID="user-account-switch-btn"
-            />
+            <React.Fragment>
+              <Button
+                text="Switch Account"
+                icon={IconFont.Switch_New}
+                onClick={showSwitchAccountDialog}
+                testID="user-account-switch-btn"
+              />
+              <hr />
+            </React.Fragment>
           )}
-          <hr />
-          <h2 data-testid="account-settings--header"> Account Details </h2>
-          <div data-testid="account-active-name--block">
-            Currently logged in Active Account: {activeAcctName}
-          </div>
 
+          <h2 data-testid="account-settings--header"> Account Details </h2>
```

Comment:
clockface prodives a `Heading` component for this: https://influxdata.github.io/clockface/?path=/story/components-heading--base